apps/openmw/mwclass/container.cpp

```diff
@@ -145,7 +145,7 @@ namespace MWClass
         // make key id lowercase
         std::string keyId = ptr.getCellRef().getKey();
         Misc::StringUtils::lowerCaseInPlace(keyId);
-        for (MWWorld::ContainerStoreIterator it = invStore.begin(); it != invStore.end(); ++it)
```

Comment:
> The only thing that bothers me, is when you try to convert ConstIterator to Iterator you get a linking error instead of a compiling error. But that keeps the code shorter.

That sounds worth fixing. How much longer would the code have to be?